CHANGES.rst

```diff
@@ -14,7 +14,7 @@ New builtins
 * ``Series``,  ``O`` and ``SeriesData``
 * ``StringReverse``
 * Add all of the named colors, e.g. ``Brown`` or ``LighterMagenta``.
-
+* ``Collect``
```

Comment:
(I've been keeping these in alphabetic order. Not a big deal though.)

Reply:
OK, when I finish cleaning this, I will put in the right place..